### DISCUSSION TITLE
X6如何实现类似xflow的JsonForm 配置式表单

### DISCUSSION THREAD

**Discussion by FiccNotes** [antvis/X6#3191] [Q&A]
类似这种：https://xflow.antv.vision/zh-CN/docs/tutorial-ext-components/canvas-config-form
点击某个节点或线后，可以修改该元素的属性

**Comment by NewByVector** [comment]
通过表单项修改节点属性：http://x6.antv.antgroup.com/tutorial/basic/node#%E4%BF%AE%E6%94%B9%E8%8A%82%E7%82%B9

**Reply by FiccNotes** [reply]
感谢！有没有类似xFlow这种更丰富的demo呢？

新手入门，上面文档的例子能看明白，但在项目中实现类似xFlow中JsonForm的更丰富的配置表单还是觉得有难度。

当用户点击Canvas图中某个元素后，显示对应的配置菜单，并在配置菜单中带入显示元素的一些属性值，不同类型的元素显示不同的菜单；

类似下面这个图中左下角这种
https://gw.alipayobjects.com/mdn/rms_19b204/afts/img/A*_-PERKth6egAAAAAAAAAAAAAARQnAQ

上图来自下面链接的  “功能介绍”->“节点交互”
https://xflow.antv.vision/zh-CN/docs/tutorial/solutions/dag/

谢谢！

**Reply by NewByVector** [reply]
这个不在 X6 的 scope 内，在 XFlow 的 scope 内，不过 XFlow 1.x 因为设计上的一些问题，现在难以维护，我们内部也在开发 2.x 版本，会带上这个功能。不过需要一段时间才能发布。
